### PR TITLE
merge dev to prod

### PR DESCRIPTION
**Pull request by thomas779**
changelog:
- [x] enabled bridge & redeem for fungibles
- [x] enabled bridge & redeem for non-fungibles
- [x] refactored file structure
- [x] enabled websocket reconnections

### CODE REVIEW

File: core/bridge_fungible.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	if !isTokenSupported {
+		// if no --> all attestToken on destination chain
+		att := attestationFromTokenTransfer(tokenTransfer, client)
+		tx, err := dstBridge.AttestToken(auth, att)
+		if err != nil {
+			fmt.Println("failed to attest token: ", err)
+			return
+		}
+		fmt.Println("attest token tx hash: ", tx.Hash())
+	}
```
Comment (thomas779) [inline core/bridge_fungible.go]: This code minted an attestation that didn't work when I ran it myself on a new ERC20/ERC721, not sure why... can you take a look @lajosdeme ?

Comment (thomas779) [reply core/bridge_fungible.go]: it failed to use that attestation to mint new tokens on the destChain

Comment (lajosdeme) [reply core/bridge_fungible.go]: yeah, the issue is that we don't wait for the `dstBridge.AttestToken` to be confirmed, and try to bridge before the wrapped token is even deployed. Pushing a fix for this soon

Comment (lajosdeme) [reply core/bridge_fungible.go]: fixed here: 88257bf7a9581265c37d3e620bf772987f4bea6f

**Pull request merged**